Readme.org

```diff
@@ -94,15 +94,15 @@ these are things you need to know.
 ** Exported stuff
 
    This extension exports a number of constants and functions to an
-   object ~global.screen.workspace_grid~ for your convenience.  (It
+   object ~Utils.WS.getWS().workspace_grid~ for your convenience.  (It
```

Comment:
@matttbe what about @felipemarinho97 comment about Utils not being a global?

https://github.com/zakkak/workspace-grid/pull/81#discussion_r214859254

Reply:
Thank you, should be fixed now